internal/datanode/flush_manager.go

```diff
@@ -683,6 +683,7 @@ func (t *flushBufferInsertTask) flushInsertData() error {
 		err := group.Wait()
 		metrics.DataNodeSave2StorageLatency.WithLabelValues(fmt.Sprint(paramtable.GetNodeID()), metrics.InsertLabel).Observe(float64(tr.ElapseSpan().Milliseconds()))
 		if err == nil {
+			log.Warn("failed to flush insert data", zap.Error(err))
```

Comment:
I don't think this is the right place to log failed to flush, its `err==nil` above